### PR TITLE
docs(helper): add missing helper docstrings

### PR DESCRIPTION
**Pull request by Yongxuanzhang**

### CODE REVIEW

File: jina/helper.py [hunk 1]
```diff
@@ -904,7 +948,10 @@ def run_async(func, *args, **kwargs):
     """
 
     class RunThread(threading.Thread):
+        """Create a running thread when in Jupyter notebook."""
```
Comment (maximilianwerk) [inline jina/helper.py:951]: This class is private to the function. Having docstrings is OK, but I'd still rename it to `_RunThread` in order to highlight, that it is private.

File: jina/helper.py [hunk 2]
```diff
@@ -904,7 +948,10 @@ def run_async(func, *args, **kwargs):
...
+
         def run(self):
+            """Run function for thread."""
```
Comment (maximilianwerk) [inline jina/helper.py:954]:
```suggestion
            """Runs given `func` asynchronously."""
```

File: jina/helper.py [hunk 3]
```diff
@@ -743,11 +757,41 @@ def typename(obj):
 
 
 def rsetattr(obj, attr: str, val):
```
Comment (maximilianwerk) [inline jina/helper.py:759]: `rsetattr` and `rgetattr` are not used anywhere in the codebase anymore. Let's remove them.

**Review by maximilianwerk — CHANGES_REQUESTED** [review]
I'd rather remove not used functions.

File: jina/helper.py (hunk 1 above)
Comment (Yongxuanzhang) [reply jina/helper.py:906]: Done

File: jina/helper.py (hunk 2 above)
Comment (Yongxuanzhang) [reply jina/helper.py]: Done

File: jina/helper.py (hunk 3 above)
Comment (Yongxuanzhang) [reply jina/helper.py]: Done

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**